internal/rhcs/cluster/clusterschema/proxy_schema.go

```diff
@@ -0,0 +1,144 @@
+package clusterschema
+
+import (
+	"fmt"
```

Comment:
I think imports should be formatted as :
go imports
external imports
internal import
I think imports should be formatted as :
go imports
external imports
internal import

at all places